packages/expo-dev-menu/android/src/test/java/unvendored/UnVendoredClass.kt

```diff
@@ -0,0 +1,3 @@
+package unvendored
+
+class UnVendoredClass
```

Comment:
😅 Also please change the file name 
```suggestion
class UnvendoredClass
```